src/bitbox02.js

```diff
@@ -635,6 +635,8 @@ export class BitBox02API {
         withdrawals,
         validityIntervalStart,
     }) {
+        // Undefined (missing) ttl is passed as ttl=0 AND allowZeroTTL=false.
+        const allowZeroTTL = ttl !== undefined;
```

Comment:
i wonder what happens if someone sets ttl to `null`. does that translate to `AllowZeroTtl` set to `false` in gowrapper/cardano.go?

Reply:
null is not a valid input according to the docs above, only string or `undefined` (will we clearer with typings).

Empty string is an edge case I missed. I added an error check for that.

I changed it to `!!ttl` so empty string also works, and elaborated in the docstring.

Reply:
the problem with semantically invalid inputs in js is it allows to pass the args anyway and makes bugs harder to find. i think it's worth adding at least a tiny bit of protection: many still use null instead of undefined in js.

we're expecting more integration and usage of bb02 apis in the future, right :)